src/services/sessions.js

```diff
@@ -861,7 +861,7 @@ module.exports = class SessionsHelper {
 				}
 			}
 
-			if (userId != sessionDetails.mentor_id) {
+			if (userId != sessionDetails.mentor_id && userId != sessionDetails.created_by) {
 				delete sessionDetails?.meeting_info?.link
```

Comment:
@nevil-mathew Why we are deleting meeting_info ?

Reply:
That's the existing flow, i just updated it to not delete those info if the creator is getting the details.